am-lib/src/test/java/uk/gov/hmcts/reform/amlib/helpers/TestDataFactory.java

```diff
@@ -28,6 +29,23 @@ public static ExplicitAccessRecord createRecord(String resourceId,
             .build();
     }
 
+    public static ExplicitAccessRecord createRecord(String resourceId,
+                                                    String accessorId,
+                                                    Set<Permission> explicitPermissions,
+                                                    String attribute) {
+        return ExplicitAccessRecord.builder()
+            .resourceId(resourceId)
+            .accessorId(accessorId)
+            .explicitPermissions(explicitPermissions)
+            .accessType(TestConstants.ACCESS_TYPE)
```

Comment:
`TestConstants.ACCESS_TYPE` I'd rename it to: `TestConstants.EXPLICIT_ACCESS_TYPE`

Maybe not in this PR, but in the future. Especially when we start using other access types.

Reply:
👍